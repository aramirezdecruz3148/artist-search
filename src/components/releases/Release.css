@import url('https://fonts.googleapis.com/css?family=Monoton|Poiret+One|Viga&display=swap');

.release {
  height: 20vw;
  width: 30vw;
  text-align: center;
  margin: 0 3vw 3vw 0;
  padding-top: 3vw;
  border: .2vw solid #AFEEEE;
  font-size: 1.4vw;
  font-family: 'Viga', sans-serif;
  font-weight: bolder;
}

.release img {
  width: 10vw;
}

.link {
  text-decoration: none;
  color: rgb(78, 78, 78);
}

.link:hover {
  color: white;
  transform: scale(1.2);
}

.release:hover {
  transform: scale(1.2);
  background: rgb(251, 147, 147);
  font-size: 1.6vw;
}