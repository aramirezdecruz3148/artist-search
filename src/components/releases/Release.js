import React from 'react';
import PropTypes from 'prop-types';

function Release() {
  return;
}

Release.propTypes = {

};

export default Release;
