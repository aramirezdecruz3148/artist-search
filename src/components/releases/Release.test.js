import React from 'react';
import { shallow } from 'enzyme';
import Release from './Release';

describe('Release component', () => {
  it('renders Release', () => {
    const wrapper = shallow(<Release releaseCover='string' releaseId='234j2h3b4j2' releaseDate='March 14th, 1988' releaseTitle='My life' />);
    expect(wrapper).toMatchSnapshot();
  });
});
