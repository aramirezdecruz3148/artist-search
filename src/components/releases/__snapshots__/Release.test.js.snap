// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Release component renders Release 1`] = `
<Fragment>
  <img
    src="string"
  />
  <Link
    to="/songs/234j2h3b4j2"
  >
    <h3>
      My life
    </h3>
  </Link>
  <h5>
    Release Date: 
    March 14th, 1988
  </h5>
</Fragment>
`;
